store/tikv/coprocessor.go

```diff
@@ -219,6 +213,50 @@ const rangesPerTask = 25000
 
 func buildCopTasks(bo *Backoffer, cache *RegionCache, ranges *copRanges, req *kv.Request) ([]*copTask, error) {
 	start := time.Now()
+	rangesLen := ranges.len()
+
+	tasksCh := make(chan *copTask, 2048)
```

Comment:
```suggestion
	tasksCh := make(chan *copTask)
```
It seems that we do not need a buffer here.

Reply:
Done